module-system/module-implementations/examples/sov-election/src/lib.rs

```diff
@@ -55,11 +59,16 @@ impl<C: sov_modules_api::Context> sov_modules_api::Module for Election<C> {
 
     type CallMessage = call::CallMessage<C>;
 
-    fn genesis(
+    type GasConfig = DefaultGasConfig;
+
+    fn genesis<G>(
         &self,
         config: &Self::Config,
-        working_set: &mut WorkingSet<C::Storage>,
-    ) -> Result<(), Error> {
+        working_set: &mut WorkingSet<C::Storage, G>,
+    ) -> Result<(), Error>
+    where
+        G: GasUnit,
```

Comment:
We have to fins a more ergonomic way to pass `GasUnit` around. My firs idea is to pot it in `C: Context` this way `WorkingSet` won't need a an explicit `G`

Reply:
Very good suggestion; onto that